document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -46,33 +50,38 @@
   private static final String CREATED_TIME = "createdTime";
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
-  private final DBCollection collection;
+  private final com.mongodb.client.MongoCollection<BasicDBObject> collection;
 
-  MongoCollection(DBCollection collection) {
+  MongoCollection(com.mongodb.client.MongoCollection<BasicDBObject> collection) {
```

Comment:
Something wrong with having this among the imports?

Reply:
Oh probably coz this class is also called MongoCollection. Got it.

Reply:
It conflicts with the class name :)